### PR TITLE
feat(euiccinfo): add euiccCiPKIdListFor{Verification,Signing}

### PR DESCRIPTION
**Pull request by CoelacanthusHex**
According to RFC3280 section 4.2.1.2, SubjectKeyIdentifier is byte array of SHA-1 or subset of SHA-1, so we print it as HEX string, just as InfiLPA does.

Ref: https://datatracker.ietf.org/doc/html/rfc3280#section-4.2.1.2

### CODE REVIEW

**Review by septs — APPROVED** [review]

File: src/applet/chip/info.c [hunk 1]
```diff
@@ -10,7 +10,7 @@ static int applet_main(int argc, char **argv)
     char *eid = NULL;
     char *default_smdp = NULL;
     char *default_smds = NULL;
-    struct es10cex_euiccinfo2 euiccinfo2;
+    struct es10cex_euiccinfo2 *euiccinfo2 = NULL;
```
Comment (estkme) [inline src/applet/chip/info.c:13]: 这里可能不太对

File: src/applet/chip/info.c [hunk 2]
```diff
@@ -26,7 +26,7 @@ static int applet_main(int argc, char **argv)
         return -1;
     }
 
-    if (es10cex_get_euiccinfo2(&euicc_ctx, &euiccinfo2) == 0)
+    if (es10cex_get_euiccinfo2(&euicc_ctx, euiccinfo2) == 0)
```
Comment (estkme) [inline src/applet/chip/info.c:29]: 这里可能不太对

**Review by estkme — CHANGES_REQUESTED** [review]

File: euicc/es10cex.h [hunk 3]
```diff
@@ -9,10 +9,19 @@ struct es10cex_euiccinfo2
     char euicc_firmware_version[16];
     char uicc_firmware_version[16];
     char global_platform_version[16];
+    struct {
+        char **list;
+        int count;
+    } euicc_ci_public_key_id_list_for_verification;
+    struct {
+        char **list;
+        int count;
```
Comment (estkme) [inline euicc/es10cex.h:18]: 这个count其实是不必须的，可以使用NULL-Terminate array list，即最后一个指针指向NULL，这也符合整个项目的风格（HTTP Interface中对header的数据结构）

**Review by estkme — CHANGES_REQUESTED** [review]

File: src/applet/chip/info.c (hunk 1 above)
Comment (CoelacanthusHex) [reply src/applet/chip/info.c]: done by alloc an object for it, see [`b1f20ad` (#29)](https://github.com/estkme-group/lpac/pull/29/commits/b1f20adfeeaba5ad813fe33bcf2958eabe90c41e)

File: euicc/es10cex.h (hunk 3 above)
Comment (CoelacanthusHex) [reply euicc/es10cex.h]: Done in [`1048f03` (#29)](https://github.com/estkme-group/lpac/pull/29/commits/1048f03350c131e68a282fbbce1f78e73383a398)

**Pull request merged**